lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -122,6 +122,16 @@ class GoogleCloudOutput < BufferedOutput
     # Cloud Logging API.
     config_param :use_grpc, :bool, :default => false
 
+    # Whether to allow non-UTF-8 characters in user logs. If set to true, any
+    # non-UTF-8 character would be replaced by the string specified by
+    # 'non_utf8_replacement'. If set to false, any non-UTF-8 character would
```

Comment:
FWIW, this is non-Rubyesque. It's preferred to leave the expected type (if any) to the documentation, rather than embed it into the name. That way the functionality can be extended later. Let's please fix this before customers start depending on this behavior.